.gitignore

```diff
@@ -10,6 +10,7 @@ node_modules/
 .idea/
 .eslintcache
 .env.local
+.env
```

Comment:
Added this while playing around with Cypress + `dotenv`.  Didn't see a reason to remove it when I got Cypress + dotenv working with our `.env.local` file.  It prevents accidental commits of secrets if someone creates a `.env` file in future.